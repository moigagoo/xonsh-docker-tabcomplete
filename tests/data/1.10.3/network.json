{
        "arguments": [
                "--help"
        ],
        "commands": [
                "connect",
                "disconnect",
                "inspect",
                "ls",
                "rm",
                "create"
        ]
}
