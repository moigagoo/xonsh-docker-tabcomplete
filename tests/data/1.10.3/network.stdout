
Usage:	docker network [OPTIONS] COMMAND [OPTIONS]

Commands:
  connect                  Connect container to a network
  disconnect               Disconnect container from a network
  inspect                  Display detailed network information
  ls                       List all networks
  rm                       Remove a network
  create                   Create a network

Run 'docker network COMMAND --help' for more information on a command.

  --help             Print usage
